examples/csi-app.yaml

```diff
@@ -11,10 +11,22 @@ spec:
       volumeMounts:
         - mountPath: "/data"
           name: my-csi-volume
-      command: [ "sleep", "1000000" ]
+      command:
+        - sh
+        - -c
+        - |
+          while true
+          do
+            ls -la /data
+            touch /data/foo
+            ls -la /data
```

Comment:
this rework is a way to prove updates to the content occur without having to `oc rsh` or `oc exec` into the pod and run `ls -la /data`